### ISSUE TITLE
Block Disbursements - $0/Ineligible Disbursement - FT/PT

### ISSUE THREAD

**Issue by CarlyCotton** [bcgov/SIMS#3416]
**User Story**
As a ministry user, I do not want to disburse any $0 awards or disbursements. This means we need to block the disbursement processes and treat this like other blocked disbursements (see #2798, #2536) with a tracker card and email. Assessments are blocked from acceptance (see #2706), but there may be a case where it was accepted previously with a >$0 award and a reassessment has caused the total award to become $0. This is why I need an additional blocker in place for these disbursements.

**Acceptance Criteria**
- [x] Do not try to send an eCert disbursement for $0 in **PART TIME OR FULL TIME.** 
- [x] This should be based as $0 at the estimated phase. 
- [x] Treat this like other blocked disbursements with a tracker card and email (Generic tracker content:  Your disbursement has been blocked as there is no amount to be disbursed.)

**Business Context**
Other related communications and notifications on #3423.

**Technical**
- [ ] Two first ACs were already implemented as part of the #3382 and the third AC is partially implemented (already blocking the disbursement) ([source code reference](https://github.com/bcgov/SIMS/blob/9039e1e4449a7b0be0723046247102eb7dcb17b9/sources/packages/backend/libs/integrations/src/services/disbursement-schedule/e-cert-processing-steps/validate-disbursement-base.ts#L57)).
- [ ] Need to add the tracker content.
- [ ] Add E2E test for the e-Cert generation to validate the email sent and blocked disbursement.
- 

Feb 13, 2025 - Additional Context.
[BUG]: The scenario was triggered in Production by manually reassessing completed application. There were no changes to data and limits were reached with the first disbursement, making student ineligible for extra funding. SIMS issued eCert despite this ticket being in place. 
Steps to  reproduce:
1. Create a new student application with the following data: 
capacity: part-time 
disability information: no
marital status: single
do you have any eligible dependants: no
total income in 2023: 5000$
transportation cost: no
2. Accept assessment as a student
3. Confirm Enrolment as Institution 
4. Disburse money (take a note of how much student is eligible for)
--------------------
5.  Once disbursement occurs, log in as a Ministry user and trigger manual reassessment without changing any other data
6.  Confirm Enrolment as Institution 
7.  Run eCert job. 
8.  BUG: The second eCert is not being blocked by system despite the fact that student not eligible for any additional funding. 
Additional context: this is currently caught by Nationals, but not by SIMS

**Comment by AnnaPBashkatova** [comment]
@dheepak-aot Hello Dheepak, I reopened this ticket as it does not function as expected in Production. I update the body of the ticket with more context to regards of actual behavior. 
Thank you, 
Anna

**Comment by ninosamson** [comment]
@AnnaPBashkatova  - would suggest new bug ticket rather than re-opening the ticket.

**Comment by AnnaPBashkatova** [comment]
@ninosamson @dheepak-aot Closing this ticket, created a new ticket #4357
